src/lib.rs

```diff
@@ -86,6 +86,10 @@ pub struct Claims {
 
     /// NATS claims
     pub nats: NatsClaims,
+
+    /// Time when the token expires (in seconds since the unix epoch)
+    #[serde(rename = "exp", skip_serializing_if = "Option::is_none")]
+    pub expires: Option<i64>,
```

Comment:
is there any reason for this to be an i64 instead of u64?

Reply:
Looks like the official go implementation uses i64 as well
https://github.com/nats-io/jwt/blob/4a8a7328f44f94e4e9d4582fbd7f439100a2cd0e/claims.go#L66

I suppose that means issued_at should probably be changed to an i64 too at some point.

For now I'll merge this as is